components/PlantItem.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { View, Text, Image} from 'react-native';
+
+import styles from '../styles/styles';
+
+export default function PlantItem (props) {
+  return(
+    <View>
+      {/* <Image style={styles.meetAPlant} source={require(props.image)} /> */}
```

Comment:
I wonder what was going on with the rendering of the image? Could it be because it is .png?